package.json

```diff
@@ -12,12 +12,12 @@
     "**/*.js"
   ],
   "scripts": {
+    "test": "jest",
     "tsc": "tsc",
-    "lint": "npm run lint:base && npm run lint:jest && npm run lint:react && npm run lint:typescript",
+    "lint": "npm run lint:base && npm run lint:jest && npm run lint:react",
     "lint:base": "eslint --report-unused-disable-directives --config ./configs/base.js .",
     "lint:jest": "eslint --report-unused-disable-directives --config ./configs/jest.js",
-    "lint:react": "eslint --report-unused-disable-directives --config ./configs/react.js",
-    "lint:typescript": "eslint --report-unused-disable-directives --config ./configs/typescript.js **/*.ts"
```

Comment:
Removed this and instead added `overrides` in eslintrc, like downstream projects do

Reply:
well the point of this was so that we can test the javascript-only rules independently from the typescript rules. so you will need to figure out a different way of doing that (if we care)

Reply:
Isn't that handled by just running `lint:base` on a .js file?

Reply:
actually you might be right. the reason I originally set it up that way was because all the typescript rules were included globally. that was before we started applying the typescript rules as an override for .ts files only.

so in that case I'm not sure why the .ts example test is giving you an error

Reply:
also yes, I'm pretty sure you can remove `--config ./configs/base.js` because that is already inherited